src/main/java/spoon/support/reflect/reference/CtTypeReferenceImpl.java

```diff
@@ -479,6 +463,13 @@ public boolean isAnonymous() {
 	}
 
 	@Override
+	public boolean isEnclosingType() {
+		final Pattern pattern = Pattern.compile("^([0-9]+)([a-zA-Z]+)$");
```

Comment:
for sake of future maintenance, explain what this means?

add a pointer to a document stating this convention for compilers? (or at least for JDT)?